<launch>
	<include ns="bridge1" file="$(find rosbridge_server)/launch/rosbridge_websocket.launch">
		<arg name="port" value="9090" />
	</include>

	<node name="team405" pkg="team405" type="team405_node" output="screen" args="_image_transport:=compressed" cwd="node">

        <param name="camera_rgb" value="/camera/rgb/image_raw" />
        <param name="camera_depth" value="/camera/depth/image_raw" />

        <param name="btn1" value="/bt1_status" />
        <param name="btn2" value="/bt2_status" />
        <param name="btn3" value="/bt3_status" />
        <param name="btn4" value="/bt4_status" />
        <param name="distance_sensor" value="/ss_status" />

        <param name="led" value="/led_status" />
        <param name="lcd_print" value="/lcd_print" />

        <param name="speed" value="/set_speed_car_api" />
        <param name="steer" value="/set_steer_car_api" />

    </node>  

    <node name="image_view" pkg="rqt_image_view" type="rqt_image_view" output="screen" />
</launch>
